packages/vector_graphics_compiler/example/README.md

```diff
@@ -0,0 +1,9 @@
+# example
+
+A Flutter showcase for how to replce asset transformers with hooks.
+
+To run execute:
+```bash
+flutter config --enable-dart-data-assets
+flutter run -d linux
```

Comment:
Are there technical reasons for only supporting this on Linux? Ideally we'd at least support the desktop platforms so that the majority of our users can run this.